Mage.Tests/src/test/java/org/mage/test/cards/conditional/SentinelSarahLyonsTest.java

```diff
@@ -0,0 +1,91 @@
+package org.mage.test.cards.conditional;
```

Comment:
Test folder tree is a bit of a mess.

I probably would have put this one in the single cards in `/test/cards/single/pip/SentinelSarahLyonsTest.java`.

Reply:
Thanks, I wasn't exactly sure where it should go.